### ISSUE TITLE
Charm submission process non-obvious for Charm Partner Program (CPP) members

### ISSUE THREAD

**Issue by randall-ubuntu** [juju/docs#228]
The current page at:
https://juju.ubuntu.com/docs/authors-charm-store.html
buries the need to open a bug too deep in the page. Our partners are consistently missing that step as it in non-obvious that they need to file a bug to get a review.

Proposal:

Change this paragraph
"Charm Store Submission

There are currently 2 methods to submit a charm and have it listed in the charm store. Both methods have their perks - but it is suggested to start with your personal namespace before asking for a charmer featured charm."

to this:

Charm Store Submission

There are currently two (2) methods to submit a Charm and to have it listed in the Charm store. They are outlined below. 

<indent & bold>
Important Note: if you are a member of the Charm Partner Program, you will want to ensure that your Charm gets into the Recommended Charms section of the Charm Store, so please ensure to follow the instructions in the "Recommended Charms" section below.
</indent & blod>

Both Charm submission methods have their perks - but it is suggested to start with your personal namespace before asking for a charmer featured charm.

**Comment by evilnick** [comment]
isn't there a page specifically for CPP? I think on insights, which should address this.
Though I will look at adding a note

**Comment by marcoceppi** [comment]
It should probably be mirrored/live here as we don't control insights and the submission process for charms is bound to change in the more near than not future.

**Comment by evilnick** [comment]
No, we don't control insights. I offered to have the charm partner page as part of the docs, but the decision was made to put it elsewhere. Having two pages which do the same thing is just confusing, and there is no certainty that if we were to make one in the docs it would receive necessary updates regarding things outside the realms of the charm store etc. I believe the insights page is being maintained so...


Anyhow, I added a note to the charm store page.
